pallets/identity/src/lib.rs

```diff
@@ -1980,6 +1980,31 @@ impl<T: Trait> Module<T> {
 }
 
 impl<T: Trait> Module<T> {
+    /// RPC call to fetch some aggregate account data for fewer round trips.
+    pub fn get_key_id_data(acc: T::AccountId) -> Option<types::KeyIdData<IdentityId>> {
```

Comment:
Can I be a pain and ask for the function to be called `get_key_information` (or `get_key_data`).

Reply:
So the reason why I called it `key_id_data` is because I think of this as a sort of "junction-table metadata" between keys & DIDs.

Reply:
We'll go with `key_identity_data`.